src/client/components/features/Logement/Annonce.tsx

```diff
@@ -52,7 +51,8 @@ export function AnnonceDeLogement(props: HitProps<AnnonceDeLogementIndexee>) {
 					key={annonce.slug}
 					className={styles.CardFooterCallToAction}
 					prefetch={false}
-					appearance={'asQuaternaryButton'}>
+					appearance={'asQuaternaryButton'}
+					aria-labelledby={`annonceTitre-${annonce.slug} Lire l‘annonce`}>
```

Comment:
Lire l'annonce est pas nécessaire le but est de récupérer l'id du titre de l'annonce. Je l'aurais également extrait dans une variable pour que le lien soit plus fort.

Reply:
Si tu veux ajouter "Lire l'annonce" pour que les lecteurs d'écran le restitue, en l'état VoiceOver ne dit que le titre de l'annonce. Il faut trouver autre chose si c'est le comportement souhaité (je veux bien voir la solution si tu l'implémentes)

Reply:
"pour que le lien soit plus fort" je ne comprend pas trop...

Reply:
Je viens de retester désolé. 
```suggestion
					aria-labelledby={`annonceTitre-${annonce.slug}`}>
```
Avec ça la restitution vocale donne déjà "Lire l'annonce <titre de l'annonce>" donc je ne pense pas qu'il soit nécessaire de le rajouter